src/v2/Apps/Purchase/Components/OrderRow.tsx

```diff
@@ -141,32 +169,40 @@ const OrderRow: React.FC<OrderRowProps> = props => {
           >
             {order.state.toLowerCase()}
           </Text>
+          {trackingId && (
+            <>
+              <Text variant="text" mx={1}>
+                &#8226;
+              </Text>
+              <Link target="_blank" href={trackingURL}>
```

Comment:
Small security/performance addition we should include on all _blank links that aren't internal. 

```suggestion
              <Link target="_blank" rel="noopener noreferrer" href={trackingURL}>
```